### ISSUE TITLE
Window logMessage - Runner connection error: Timed out connecting to [::]:54532

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge-vscode#131]
**Expected behavior**
Should not get error `Runner connection error: Timed out`

**Actual behavior**
Getting the above error on opening the gauge-ruby project

**Steps to replicate**
```
17:17:15.878 --> request #0: initialize: {"processId":16228,"rootPath":"c:\\work\\testFolder\\ruby\\basic","rootUri":"file:///c%3A/work/testFolder/ruby/basic","capabilities":{"workspace":{"applyEdit":true,"didChangeConfiguration":{"dynamicRegistration":true},"didChangeWatchedFiles":{"dynamicRegistration":true},"symbol":{"dynamicRegistration":true},"executeCommand":{"dynamicRegistration":true}},"textDocument":{"synchronization":{"dynamicRegistration":true,"willSave":true,"willSaveWaitUntil":true,"didSave":true},"completion":{"dynamicRegistration":true,"completionItem":{"snippetSupport":true,"commitCharactersSupport":true}},"hover":{"dynamicRegistration":true},"signatureHelp":{"dynamicRegistration":true},"definition":{"dynamicRegistration":true},"references":{"dynamicRegistration":true},"documentHighlight":{"dynamicRegistration":true},"documentSymbol":{"dynamicRegistration":true},"codeAction":{"dynamicRegistration":true},"codeLens":{"dynamicRegistration":true},"formatting":{"dynamicRegistration":true},"rangeFormatting":{"dynamicRegistration":true},"onTypeFormatting":{"dynamicRegistration":true},"rename":{"dynamicRegistration":true},"documentLink":{"dynamicRegistration":true}}},"trace":"off"}
17:17:15.882 <-- result #0: initialize: {"capabilities":{"textDocumentSync":1,"completionProvider":{"resolveProvider":true,"triggerCharacters":["*","* ","\"","\u003c",":",","]},"definitionProvider":true,"documentSymbolProvider":true,"workspaceSymbolProvider":true,"codeActionProvider":true,"codeLensProvider":{},"documentFormattingProvider":true,"renameProvider":true}}
17:17:15.892 --> notif: initialized: {}
17:17:16.994 --> request #1: gauge/specs: {}
17:17:17.020 <-- result #1: gauge/specs: [{"heading":"Specification Heading","executionIdentifier":"c:\\work\\testFolder\\ruby\\basic\\specs\\example.spec"}]
17:17:45.951 <-- notif: window/logMessage: {"type":4,"message":"Runner connection error: Timed out connecting to [::]:54532"}
17:17:45.953 <-- request #0: client/registerCapability: {"registrations":[{"id":"gauge-runner-didOpen","method":"textDocument/didOpen","registerOptions":{"documentSelector":{"scheme":"file","language":"ruby","pattern":"c:\\work\\testFolder\\ruby\\basic/**/*"}}},{"id":"gauge-runner-didClose","method":"textDocument/didClose","registerOptions":{"documentSelector":{"scheme":"file","language":"ruby","pattern":"c:\\work\\testFolder\\ruby\\basic/**/*"}}},{"id":"gauge-runner-didChange","method":"textDocument/didChange","registerOptions":{"documentSelector":{"scheme":"file","language":"ruby","pattern":"c:\\work\\testFolder\\ruby\\basic/**/*"},"syncKind":1}},{"id":"gauge-runner-codelens","method":"textDocument/codeLens","registerOptions":{"documentSelector":{"scheme":"file","language":"ruby","pattern":"c:\\work\\testFolder\\ruby\\basic/**/*"}}}]}
17:17:45.962 --> result #0: client/registerCapability: null
17:17:46.257 <-- notif: textDocument/publishDiagnostics: {"uri":"file:///c%3A/work/testFolder/ruby/basic/specs/example.spec","diagnostics":[]}
17:19:15.745 --> request #2: shutdown: null
17:19:15.747 <-- result #2: shutdown: null
```

**Version**
```
Gauge version: 0.9.8.nightly-2018-02-02
Commit Hash: bff4f46

Plugins
-------
ruby (0.4.3.nightly-2018-01-18)
```

**Comment by sguptatw** [comment]
**Manual steps to replicate**
* Keep only one ruby implementation file open. (Ensure all other files are closed)
* Reload the project

**Comment by sguptatw** [comment]
Fixed in version - Gauge version: 0.9.8.nightly-2018-02-14
